Charm/ApplicationCore.h

```diff
@@ -64,7 +64,7 @@ class ApplicationCore : public QObject
         ShowAndRaise
     };
     explicit ApplicationCore( TaskId startupTask, QObject* parent = nullptr );
-    ~ApplicationCore();
+    ~ApplicationCore() override;
```

Comment:
We discussed it on all, while there was no consensus for override (most find it noise), we figured that it's not that bad either and this makes all tools shut up.